### PR TITLE
update gradle version to 2.5

### PR DESCRIPTION
**Pull request by sqreek**
new version come with nice [feature](https://docs.gradle.org/current/release-notes#continuous-build) for TDD developers

### CODE REVIEW

**Comment by sqreek** [comment]
ping @Wikia/services-team

**Comment by pchojnacki** [comment]
LGTM

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Mon Feb 16 17:33:10 CET 2015
+#Thu Jul 09 16:02:04 CEST 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.5-bin.zip
```
Comment (pchojnacki) [inline gradle/wrapper/gradle-wrapper.properties]: try with gradle-2.5-all first

Comment (sqreek) [inline gradle/wrapper/gradle-wrapper.properties]: I was tried, but intellij, always return it to 2.5-bin, after gradlew wrapper

**Pull request merged**